upp/stages/resampling.py

```diff
@@ -141,9 +141,15 @@ def run_on_region(self, components, region):
 
         # groupby samples
         for sample, cs in components.groupby_sample():
+            # make sure all tags equal_jets are the same
+            equal_jets_flags = [c.equal_jets for c in cs]
+            if not all(flag == equal_jets_flags[0] for flag in equal_jets_flags):
```

Comment:
For the future a cleaner way of checking this is to do `len(set(my_list)) == 1`

Reply:
I can change it now if it wasn't merged yet

Reply:
Go for it !

Reply:
done :)